src/tests/conformance/cmd/squelch.rs

```diff
@@ -28,26 +28,29 @@ use crate::{
         codecs::binary::{BinaryMessage, Payload},
         proto::{TmProposeSet, TmSquelch},
     },
-    setup::node::{Node, NodeType},
+    setup::{
+        constants::STATEFUL_NODES_COUNT,
+        node::{Node, NodeType},
+    },
     tools::{rpc::wait_for_state, synth_node::SyntheticNode},
 };
 
+// Time we shall wait for a TmProposeLedger message.
+const WAIT_MSG_TIMEOUT: Duration = Duration::from_secs(5);
```

Comment:
Can we use the other timeout constant? Or does it have to be different value (I think it was 10 secs there)?